bluetooth_mesh/messages/silvair/debug.py

```diff
@@ -218,11 +214,7 @@ class DebugSubOpcode(IntEnum):
 ArapListContentStatus = Struct(
     "current_page" / Int8ul,
     "last_page" / Int8ul,
-    "nodes" / DictAdapter(
-        GreedyRange(ArapNode),
-        key=this.address,
-        value=[this.ivi, this.sequence]
-    )
+    "nodes" / GreedyRange(ArapNode)
```

Comment:
This breaks "nice" dictionaries, as you've seen in tests. Maybe there is a way to avoid this?

Reply:
It is not possible to keep it in this "nice" format because it is not compatible with `CapNProto` which expects `List[ArapNode]`. If we want convert messages between formats we need to keeps it compatible, or create some adaptation layer which will convert `nice` messages back to "ugly" format compatible with `CapNProto`

Reply:
Ok, got it.